index.html

```diff
@@ -3413,10 +3416,127 @@ <h3>Receiving and parsing content</h3>
         Set |record|.[[\PayloadData]] to the
         <a>byte sequence</a> of |ndefRecord|'s <a>PAYLOAD field</a>.
       </li>
-    </ol>
+      <li>
+        return |record|.
+      </li>
+    </ol>  <!-- parsing NDEF smart-poster record -->
   </p>
   </section>
 
+  <section><h3>Parsing NDEF MIME type records</h3>
+    <p>
+      To <dfn>parse a NDEF MIME type record</dfn> given a |ndefRecord| into a
+      |record:NDEFRecord|, run these steps:
+      <ol class=algorithm>
+        <li>
+          Let |MIME type| be the <a>MIME type</a>
```

Comment:
Let's call it mimeType to not confuse with the <a>MIME type</a> and because of consistency

Reply:
We can do in a follow up